types/node/index.d.ts

```diff
@@ -2331,14 +2331,27 @@ declare module "child_process" {
     export interface SpawnSyncOptionsWithBufferEncoding extends SpawnSyncOptions {
         encoding: string; // specify `null`.
     }
-    export interface SpawnSyncReturns<T> {
-        pid: number;
-        output: string[];
-        stdout: T;
-        stderr: T;
-        status: number;
-        signal: string;
-        error: Error;
+    export type SpawnSyncOptions = SpawnSyncReturns.WithError | SpawnSyncOptions.WithoutError<T>;
```

Comment:
this should be `SpawnSyncReturns` consistently

Reply:
Oh, my bad, I chose wrong options in my autocomplete menu. I'll fix this ASAP.

Reply:
Done.

Reply:
Damn, I always make mistakes.